js/popup.js

```diff
@@ -461,6 +463,11 @@ function populateSubNav() {
         $("#incognito").html('<span class="green">Eyebrowse On</span>');
     }
 
+    if (user.getTickerDisplay() == true) {
+        $("#tickerDisplay").html('<span>Ticker Mode</span>');
```

Comment:
I would make a function to wrap this so you don't have to repeat it again on line 469/473